BinariesCheck.py

```diff
@@ -321,6 +321,7 @@ def __init__(self, pkg, path, fname, is_ar, is_shlib):
 srcname_regex = re.compile(r'(.*?)-[0-9]')
 invalid_dir_ref_regex = re.compile(r'/(home|tmp)(\W|$)')
 ocaml_mixed_regex = re.compile(r'^Caml1999X0\d\d$')
+usr_arch_share_regex = re.compile(r'/share/.*/(?:x86|i.86|x86_64|ppc|ppc64|s390|s390x|ia64|m68k|arm|aarch64|mips|riscv)')
```

Comment:
mips can 32-bit or 64-bit, riscv can be 32-bit, 64-bit, or 128-bit.

Reply:
Wonder if it is worth it, basically it is relevant just for power and s390 all the others are sane and do not use /usr/share/archstuff/